setup/README.adoc

```diff
@@ -0,0 +1,85 @@
+= Dev Sandbox Setup
+
+This document describes how to setup a Dev Sandbox environment that is similar to production and can be used as part of an onboarding process for new operators.
+
+=== Prereqs
+
+1. Provision an OCP cluster on AWS with sufficient resources: 4xlarge master nodes and at least 3 2xlarge worker nodes.
+
+For example, if using the openshift-install tool to install OCP on AWS you can use an install-config.yaml file that has the following configuration:
+
+----
+apiVersion: v1
+baseDomain: devcluster.openshift.com
+compute:
```

Comment:
Isn't this `compute:` redundant? There is another on at line 27.

Reply:
You're right, sorry about that. I'll fix it.

Reply:
Done 8f838bac9640657be1d14ae9151fa13b8da1bb69 Thanks, nice catch!